packages/api/tests/lambdas/sf-event-sqs-to-db-records/test-write-granules.js

```diff
@@ -549,13 +556,17 @@ test.serial('writeGranules() saves granule records to DynamoDB/PostgreSQL/Elasti
   });
 
   const dynamoRecord = await granuleModel.get({ granuleId });
-  const pgRecord = await t.context.granulePgModel.get(knex, { granule_id: granuleId });
-  t.is(pgRecord.created_at.getTime(), dynamoRecord.createdAt);
-  t.is(pgRecord.updated_at.getTime(), dynamoRecord.updatedAt);
+  const granulePgRecord = await t.context.granulePgModel.get(knex, { granule_id: granuleId });
+
+  // translate the PG granule to API granule to directly compare to Dynamo
+  const translatedPgRecord = await translatePostgresGranuleToApiGranule({
+    granulePgRecord,
+    knexOrTransaction: knex,
+  });
+  t.deepEqual(omit(translatedPgRecord, omitList), omit(dynamoRecord, omitList));
 
   const esRecord = await t.context.esGranulesClient.get(granuleId);
-  t.is(pgRecord.created_at.getTime(), esRecord.createdAt);
-  t.is(pgRecord.updated_at.getTime(), esRecord.updatedAt);
+  t.deepEqual(omit(translatedPgRecord, omitList), omit(esRecord, omitList));
```

Comment:
The idea here is to more directly compare the translated PG record to the Dynamo and ES records. It was helpful to fix the errors I was seeing in the `CreateReconcilliationReportSpec`

Reply:
Previously we were returning the ES result without modification so this effectively compares the translated PG record to the expected API response.